src/content/get-started/data-and-json.md

```diff
@@ -0,0 +1,481 @@
+
+### Task 2: Configure a Dart workspace
+
+Dart workspaces allow you to
+manage multiple related packages within a single project,
+simplifying dependency management and local development.
+Now that you're adding your third package,
+it's a good time to configure your project to use a Dart workspace.
+
+1.  **Create the root `pubspec.yaml` file.**
+
+    Navigate to the root directory of your project (`/dartpedia`) and create a
+    new file named `pubspec.yaml` with the following content:
+
+    ```yaml
+    name: _
+    publish_to: none
+
+    environment:
+      sdk: ^3.8.1 # IMPORTANT: Adjust this to match your Dart SDK version or a compatible range
+    workspace:
+      - cli
+      - command_runner
+      - wikipedia
+    ```
+
+1.  **Add workspace resolution to sub-packages.**
+
+    For each of your sub-packages (`cli`, `command_runner`, and `wikipedia`),
+    open their respective `pubspec.yaml` files and
+    add `resolution: workspace` to `pubspec.yaml`.
+    This tells Dart to resolve dependencies within the workspace.
+
+    *   For `cli/pubspec.yaml`:
+
+        ```yaml highlightLines=5
+        # ... (existing content) ...
+        name: cli
+        description: A sample command-line application.
+        version: 1.0.0
+        resolution: workspace # Add this line
+        # ... (existing content) ...
+        ```
+
+    *   For `command_runner/pubspec.yaml`:
+
+        ```yaml highlightLines=5
+        # ... (existing content) ...
+        name: command_runner
+        description: A starting point for Dart libraries or applications.
+        version: 1.0.0
+        resolution: workspace # Add this line
+        # ... (existing content) ...
+        ```
+
+    *   For `wikipedia/pubspec.yaml`:
+
+        ```yaml highlightLines=5
+        # ... (existing content) ...
+        name: wikipedia
+        description: A sample command-line application.
+        version: 1.0.0
+        resolution: workspace # Add this line
+        # ... (existing content) ...
+        ```
+
+### Task 3: Create the Summary class
+
+The Wikipedia API returns a JSON object containing a summary of an article.
+Let's create a Dart class to represent this summary.
+
+1.  Create the directory `wikipedia/lib/src/model`.
+
+    ```bash
+    mkdir -p wikipedia/lib/src/model
+    ```
+
+1.  Create the file `wikipedia/lib/src/model/summary.dart`.
+
+1.  Add the following code to `wikipedia/lib/src/model/summary.dart`:
+
+    ```dart title="wikipedia/lib/src/model/summary.dart"
+    import 'title_set.dart';
+
+    class Summary {
+      /// Returns a new [Summary] instance.
+      Summary({
+        required this.titles,
+        required this.pageid,
+        required this.extract,
+        required this.extractHtml,
+        required this.lang,
+        required this.dir,
+        this.url,
+        this.description,
+      });
+
+      ///
+      TitlesSet titles;
+
+      /// The page ID
+      int pageid;
```

Comment:
Even if this is `pageid` from the API JSON, do consider renaming the Dart class member to `pageId` to be more idiomatic Dart.